.travis.yml

```diff
@@ -1,5 +1,10 @@
 language: csharp
-sudo: false
+sudo: required
+dist: trusty
```

Comment:
But between now and then Travis builds will fail even more frequently won't they?